.github/workflows/markdown.yml

```diff
@@ -22,4 +22,5 @@ jobs:
       uses: docker://avtodev/markdown-lint:v1 # fastest way
       with:
         config: './assets/markdown/markdownlint.yml'
+        config: 'markdown/markdownlint.yml'
```

Comment:
Hi @anshuman35,

Can you elaborate please the reasoning behind this change ? I see that you added a duplicate `config` line to markdown-lint which doesn't make sense to me. The original configuration was tested and it works as far as I know.

Thanks.